### PR TITLE
jsx error

### PR DESCRIPTION
**Pull request by zairigimad**
**What kind of change does this PR introduce?**

**Did you add tests for your changes?**

**If relevant, did you update the documentation?**

**Summary**

**Does this PR introduce a breaking change?**

**Other information**

### CODE REVIEW

**Comment by arpitBhalla** [comment]
Hey @zairigimad, make changes in this file, https://github.com/react-native-elements/react-native-elements/blob/next/website/docs/switch.md

File: website/versioned_docs/version-3.4.1/switch.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ import { Switch } from 'react-native-elements';
 ```js
 <Switch value={true} />
 
-<Switch value={false} color="orange">
+<Switch value={false} color="orange"/>
```
Comment (arpitBhalla) [inline website/versioned_docs/version-3.4.1/switch.md:26]: Make change in, https://github.com/react-native-elements/react-native-elements/blob/next/website/docs/switch.md

**Review by arpitBhalla — CHANGES_REQUESTED** [review]

File: website/versioned_docs/version-3.4.1/switch.md (hunk 1 above)
Comment (arpitBhalla) [reply website/versioned_docs/version-3.4.1/switch.md:26]: Linked PR #3041

**Comment by flyingcircle** [comment]
These docs won't last too long as they'll be replaced with the correction in the next version. We'll just wait til then

**Pull request closed without merging**